### ISSUE TITLE
Silver Edge is not getting dispelled !

### ISSUE THREAD

**Issue by AvatarMagi** [ValveSoftware/Dota2-Gameplay#12923]
### Item name

Silver Edge

### Description

BKB will dispel the silver edge effect (BREAK) but the following items will not:
Satanic
Manta Style
Lotus Orb
EUL
Disperser
all of them are Basic Dispel, just like bkb.

### Example Match ID and Timestamp

Demo Hero

**Comment by Aluerie** [comment]
No. bkb doesn't dispel it. Per 733 it provides debuff immunity which interrupts silver edge debuff.
